src/main/resources/js/constants/PanelConstants.js

```diff
@@ -0,0 +1,8 @@
+const constants = {
```

Comment:
This seems suspicious, I'm guessing we'll eventually use some notion of grid rows instead of hardcoding sizes?